README.md

```diff
@@ -0,0 +1,556 @@
+  | 2.9            | yes (Python3) (last supported version for Python 2.7: 0.12.2)), Must: `pip install "setuptools>=44.1.0,<71"`   |
+  | 2.10           | yes                                                                                                            |
+  | 2.11           | yes                                                                                                            |
+
+## Installation
+
+To install XLoader:
+
+1.  Activate your CKAN virtual environment, for example:
+
+        . /usr/lib/ckan/default/bin/activate
+
+2.  Install the ckanext-xloader Python package into your virtual
+    environment:
+
+        pip install ckanext-xloader
+
+3.  Install dependencies:
+
+        pip install -r https://raw.githubusercontent.com/ckan/ckanext-xloader/master/requirements.txt
+        pip install -U requests[security]
+
+4.  Add `xloader` to the `ckan.plugins` setting in your CKAN config file
+    (by default the config file is located at
+    `/etc/ckan/default/production.ini`).
+
+    You should also remove `datapusher` if it is in the list, to avoid
+    them both trying to load resources into the DataStore.
+
+    Ensure `datastore` is also listed, to enable CKAN DataStore.
+
+5.  Starting CKAN 2.10 you will need to set an API Token to be able to
+    execute jobs against the server:
+
+        ckanext.xloader.api_token = <your-CKAN-generated-API-Token>
+
+6.  If it is a production server, you'll want to store jobs info in a
+    more robust database than the default sqlite file. It can happily
+    use the main CKAN postgres db by adding this line to the config, but
+    with the same value as you have for `sqlalchemy.url`:
+
+        ckanext.xloader.jobs_db.uri = postgresql://ckan_default:pass@localhost/ckan_default
+
+    (This step can be skipped when just developing or testing.)
+
+7.  Restart CKAN. For example if you've deployed CKAN with Apache on
+    Ubuntu:
+
+        sudo service apache2 reload
+
+8.  Run the worker:
+
+        ckan -c /etc/ckan/default/ckan.ini jobs worker
+
+## Config settings
+
+Configuration:
+
+
+This plugin supports the [`ckan.download_proxy`](https://docs.ckan.org/en/latest/maintaining/configuration.html#ckan-download-proxy) setting,
+to use a proxy server when downloading files. This setting is shared
+with other plugins that download resource files, such as
+ckanext-archiver. Eg:
+
+    ckan.download_proxy = <http://my-proxy:1234/>
+
+You may also wish to configure the database to use your preferred date
+input style on COPY. For example, to make
+[PostgreSQL](<https://www.postgresql.org/docs/current/runtime-config-client.html#RUNTIME-CONFIG-CLIENT-FORMAT>)
+expect European (day-first) dates, you could add to `postgresql.conf`:
+
+    datestyle=ISO,DMY
+
+All configurations below are defined in the 
+[config_declaration.yaml](ckanext/xloader/config_declaration.yaml) file.
+
+
+#### ckanext.xloader.jobs_db.uri
+
+Default value: `sqlite:////tmp/xloader_jobs.db`
```

Comment:
can you give another example from original rst.

``ckanext.xloader.jobs_db.uri = postgresql://ckan_default:pass@localhost/ckan_default``